code/datums/components/hide_weather_planes.dm

```diff
@@ -0,0 +1,116 @@
+/// Component that manages a list of plane masters that are dependent on weather
+/// Force hides/shows them depending on the weather activity of their z stack
+/// Applied to the plane master group that owns them
```

Comment:
you should update this to match your comment style and better describe its current job